### PR TITLE
UG-936 Display notification after unsaving article

### PR DESCRIPTION
**Pull request by Shamarni96**
### Description
After a user unsaves an article, a notification is displayed. See [designs](https://www.figma.com/file/09mlE9M0XVG5fIZy99tls2/(Final)-Professor-Engagement?node-id=758%3A7466) (Frame 245 & Frame 249).

### Ticket
https://financialtimes.atlassian.net/browse/UG-936

### How To Test
1) clone `next-article` and run `npm install build` 
2) Make sure you’re logged in on Vault and run `next-article` with `npm start`
3) Switch on the manageArticleLists flag on [Toggler](https://toggler.ft.com/#manageArticleLists)
4) visit https://local.ft.com:5050/
5) Go to any article and click the “save” button in the shareNav menu. The save article modal should appear next to the ShaveNav menu.
6) click the save button again to unsave the article. the unsave article notification should show with the text `"Removed from saved articles in myFT"` and should disappear after 5 seconds.
7) The unsaved notification should only show when the `manageArticleLists` flag is switched on. To ensure that this works, switch the `manageArticleLists` flag off and repeat steps 4 - 6

### CODE REVIEW

File: myft/ui/lists.js [hunk 1]
```diff
@@ -210,6 +221,34 @@ function initialEventListeners () {
...
+	const parentNode = document.querySelector(parentSelector);
+
+	// We're not supporting multiple notifications for now
+	// If a notification is present, we'll silently avoid showing another
+	if (document.querySelector('.myft-notification') || !parentNode) {
+		return;
+	}
+
+	const content = `
+		<p role="alert">Removed from <a href="https://www.ft.com/myft/saved-articles">saved articles</a> in myFT only.</p>
```
Comment (jamesnicholls) [inline myft/ui/lists.js:235]: **chore:** Can you check the wording with Dilyana and Ajai? I think Dilyana wanted to change it.

File: myft/main.scss [hunk 2]
```diff
@@ -323,4 +323,45 @@ $spacing-unit: 20px;
...
+			box-sizing: border-box;
+			display: flex;
+			align-items: center;
+			justify-content: center;
+			position: absolute;
+			top: 175px;
+			width: 340px;
+			height: 44px;
+			left: 50px;
+			font-size: 16px;
```
Comment (jamesnicholls) [inline myft/main.scss:338]: **nitpick:** I think you'll need to specify the font-family here as well. In the designs, the font used is Metric, but I don't think the notification is using that font locally.

File: myft/main.scss [hunk 3]
```diff
@@ -323,4 +323,45 @@ $spacing-unit: 20px;
...
+	}
+
+	.share-nav__horizontal {
+		.myft-notification {
+			background: oColorsByName('white-80');
+			box-sizing: border-box;
+			display: flex;
+			align-items: center;
+			justify-content: center;
+			position: absolute;
```
Comment (jamesnicholls) [inline myft/main.scss:354]: **suggestion (non-blocking):** You could move the styles shared between the horizontal and vertical versions of the notification into a common rule set.

File: myft/main.scss [hunk 4]
```diff
@@ -323,4 +323,45 @@ $spacing-unit: 20px;
...
+			top: 175px;
+			width: 340px;
+			height: 44px;
+			left: 50px;
+			font-size: 16px;
+			border-radius: 5px;
+
+			p {
+				margin: auto;
```
Comment (jamesnicholls) [inline myft/main.scss:342]: **question:** What is this rule for?

File: myft/ui/lists.js (hunk 1 above)
Comment (jamesnicholls) [inline myft/ui/lists.js:235]: **praise:** Nice work thinking about accessibility here. The notification is announced perfectly in VoiceOver.

**Review by jamesnicholls — CHANGES_REQUESTED** [review]
The code works perfectly, as described. I think the only blocker is that the font used in the notification doesn't look right. In the design it's Metric. Also, can you check the message in the notification with Dilyana and Ajai?

Apart from that, I have a couple of questions and suggestions.

File: myft/main.scss (hunk 4 above)
Comment (Shamarni96) [reply myft/main.scss:342]: I forgot to remove this once I figured out I didn't need it but I added it to try to center the text in the notification. I have now removed it 👍

File: myft/ui/notification.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
+let isShowing = false;
```
[the remark this answers is not in the document]
Comment (Shamarni96) [reply myft/ui/notification.js:1]: yep you are right. I have deleted this file 👍

**Review by jamesnicholls — APPROVED** [review]
Looks good, nice work.

File: myft/main.scss [hunk 6]
```diff
@@ -16,6 +16,8 @@ $n-notification-is-silent: false !default;
 @import '../components/pin-button/main';
 @import '../components/instant-alert/main';
 
+@include oTypography();
```
Comment (fenglish) [inline myft/main.scss:19]: Would this cause the same issue in the n-myft-ui's consumer apps as [this](https://github.com/Financial-Times/n-myft-dropdown/pull/8)?
I'm not 100% sure this is ok or not. Please talk to @meel-io.

Comment (Shamarni96) [reply myft/main.scss]: I have removed this line of code 👍

**Review by fenglish — APPROVED** [review]
@Shamarni96 Thank you for your change. Please keep eye on the consumer apps (not only next-article) when you release changes.

**Pull request merged**